fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -585,10 +724,124 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 			TraceEvent("RocksDB").detail("Path", a.path).detail("Method", "Close");
 			a.done.send(Void());
 		}
+
+		struct CheckpointAction : TypedAction<Writer, CheckpointAction> {
+			CheckpointAction(const CheckpointRequest& request) : request(request) {}
+
+			double getTimeEstimate() const override { return SERVER_KNOBS->COMMIT_TIME_ESTIMATE; }
```

Comment:
why is this a knob value?

Reply:
I copied over from other *Action within this file, since I don't have a better estimation. In the future I might have a better idea about the time needed for creating a checkpoint.